src/frequenz/lib/__init__.py

```diff
@@ -1,13 +1,10 @@
 # License: MIT
 # Copyright © 2023 Frequenz Energy-as-a-Service GmbH
 
-"""checker to ensure correct datetime usage with aware objects in datetime module.
+"""checker to ensure correct datetime usage with aware objects in datetime module."""
```

Comment:
While you are at it, maybe capitalize it?

 ```suggestion
"""Checker to ensure correct datetime usage with aware objects in datetime module."""
```